spec/features/view_event_spec.rb

```diff
@@ -17,35 +17,39 @@
       end
 
       scenario 'a student cannot RSVP if they are not logged in' do
-        click_on 'Attend as a student'
-
-        expect(page).to have_content(I18n.t('notifications.not_logged_in'))
+        expect(page).to have_link('Log in')
+        expect(page).to have_link('Sign up')
+        expect(page).to_not have_link('Attend as a student')
```

Comment:
In order to view an event, you have to login or sign up first. I updated the tests to reflect this change.